apps/parsley/src/components/DetailsMenu/DetailsMenuCard/Toggles/JumpToFailingLineToggle/index.tsx

```diff
@@ -16,19 +20,39 @@ const JumpToFailingLineToggle: React.FC<JumpToFailingLineToggleProps> = ({
   const { sendEvent } = usePreferencesAnalytics();
   const { logMetadata } = useLogContext();
   const isTaskLog = logMetadata?.logType === LogTypes.EVERGREEN_TASK_LOGS;
-
+  const { closeSecondGuideCue, isOpenSecondGuideCue, setIsOpenSecondGuideCue } =
+    useSectionsFeatureDiscoveryContext();
+  const triggerRef = useRef<HTMLDivElement>(null);
   return (
-    <BaseToggle
-      data-cy="jump-to-failing-line-toggle"
-      disabled={!isTaskLog}
-      label="Jump to Failing Line"
-      onChange={(value) => {
-        updateSettings({ jumpToFailingLineEnabled: value });
-        sendEvent({ name: "Toggled jump to failing line", on: value });
-      }}
-      tooltip="Automatically scroll to the failing log line on page load. Only available for task logs."
-      value={checked}
-    />
+    <>
+      {releaseSectioning && (
+        <GuideCue
+          beaconAlign="right"
+          data-cy="sections-cue-2"
+          numberOfSteps={1}
+          onPrimaryButtonClick={closeSecondGuideCue}
+          open={isOpenSecondGuideCue}
+          refEl={triggerRef}
+          setOpen={setIsOpenSecondGuideCue}
+          title="Jump to Failing Line"
+        >
+          Combined with sectioning, jump to failing line allows to streamline
+          your failure triage process.
+        </GuideCue>
```

Comment:
[nit] Slight preference to define these guide cue components in the context directory and use them here with `forwardRef` so that the code is more centralized and easily deleted